src/components/DragAndDropExample.js

```diff
@@ -0,0 +1,280 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  PanResponder,
+  Animated,
+  Dimensions
+} from 'react-native';
+import { RoundText } from './Components';
+
+const DOT_RADIUS = 5;
+const Window = Dimensions.get('window');
+
+const styles = StyleSheet.create({
+  textContainer: {
+    backgroundColor: '#1abc9c',
+    margin: 5,
+    borderWidth: 1
+  },
+  dot: {
+    backgroundColor: 'white',
+    width: DOT_RADIUS * 2,
+    height: DOT_RADIUS * 2,
+    borderRadius: DOT_RADIUS
+  }
+});
+
+// NOTE: RoundText doesn't work well with StyleSheet (calling path on props)
+const textStyle = {
+  margin: 5,
+  textAlign: 'center',
+  color: '#fff',
+  fontSize: 20
+};
+
+const sentence =
+  'conveying a statement, question, exclamation, or command, and consisting of a main clause';
+
+export default class DragAndDropExample extends React.Component {
+  constructor(props) {
+    super(props);
+    // TODO: take sentence from props later, reinit these values on prop change
+    // NOTE: there is a number of arrays, every item (word) represented there
+    // by index (so all arrays are connected by index), maybe it won't be
+    // a bad idea to create one array of objects with parameters instead
```

Comment:
Might make things a little bit easier if you can work with the objects as good as those multiple arrays.